website/sidebars.json

```diff
@@ -45,6 +46,7 @@
       "testing/testing",
       "testing/cli_testing_frameworks",
       "testing/cli_testing_selenium",
+      "testing_serenity_bdd",
```

Comment:
Looks like this should be `/testing/testing_serentity_bdd`.

Reply:
oh yeah...i will modify